test/unit/Encoding/JoseEncoderTest.php

```diff
@@ -148,7 +148,7 @@ public function base64UrlDecodeMustRaiseExceptionWhenInvalidBase64CharsAreUsed()
         $this->expectExceptionCode(0);
         $this->expectExceptionMessage('Error while decoding from Base64Url, invalid base64 characters detected');
 
-        $decoder->base64UrlDecode('áááááá');
+        $decoder->base64UrlDecode('ááá');
```

Comment:
So, `áááááá` is a valid `SODIUM_BASE64_VARIANT_URLSAFE_NO_PADDING` string, where `ááá` is not.

Currently no idea why, but I trust `ext-sodium` much more than `base64_decode`.